bp/_io.pyx

```diff
@@ -148,16 +148,16 @@ cpdef parse_newick(unicode data):
         np.ndarray[np.double_t, ndim=1] lengths
         np.ndarray[np.int32_t, ndim=1] edges
 
-    if data.count(',') == 0:
-        raise ValueError("Only trees with more than 1 node supported")
-
     data = data.strip()
     if not data.endswith(';'):
         raise ValueError("Newick does not appear terminated with a semicolon")
 
     datalen = len(data)
     topology = _newick_to_bp(data)
 
+    if len(topology.B) <= 1:
```

Comment:
This is not correct, a single node would be `[1, 0]` in the structure. Adjusted on the subsequent PR which includes the relevant changes here